pkg/api/protobuf/go/minder/v1/validators.go

```diff
@@ -17,11 +17,15 @@ package v1
 import (
 	"errors"
 	"fmt"
+	"regexp"
 )
 
 var (
 	// ErrValidationFailed is returned when a profile fails validation
 	ErrValidationFailed = fmt.Errorf("validation failed")
+
+	// Starts with a letter/digit, then a string of letter/digit and hypen or underscore
+	dnsStyleNameRegex = regexp.MustCompile(`^[a-zA-Z0-9][-_a-zA-Z0-9]{0,62}$`)
```

Comment:
Probably add a second group here to disallow names  ending in a dash or underscore:

```suggestion
	dnsStyleNameRegex = regexp.MustCompile(`^[a-zA-Z0-9]$|^[a-zA-Z0-9][-_a-zA-Z0-9]{0,62}[a-zA-Z0-9]$`)
```

Reply:
Done, though I used a non-capturing group instead of alternation.